src/defaults.json

```diff
@@ -87,13 +87,9 @@
 		"children": "Popup content",
 		"padding": "M"
 	},
-	"PopperWrapper": {
-		"children": "{<Button label=\"Click me!\" />}",
-		"popper": "{<Popup padding=\"M\">Popup content</Popup>}"
-	},
-    "ProgressBar": {
-        "percentage": 20
-    },
+  "ProgressBar": {
+      "percentage": 20
+  },
```

Comment:
Removed the demo props for PopperWrapper because it is not possible to include JS functions here (`children` and `popper` props are now render props; no longer React nodes)